lib/client.js

```diff
@@ -287,6 +287,14 @@ class AssetComputeClient extends EventEmitter {
                 }
             });
             this.eventEmitter.on('error', error => self.emit("error", error));
+
+            // NUI-878 protect against UnhandledPromiseRejectionWarning and polling interruption by having a listener by default
```

Comment:
This is in a public repo... Do we really need to keep the issue number here?

Reply:
Hmm, this is just an issue id, and we regularly post links to internal PRs etc. here, as well as the opensource CI on PRs pointing to our internal circleci.